### ISSUE TITLE
Ziro Dark Purple : Button to skip adding an existing song to playlist does not show

### ISSUE THREAD

**Issue by nishant-nayak** [spicetify/spicetify-themes#541]
**Describe the bug**
The button for *not* adding a particular song to a playlist (because the song already exists in the playlist) does not show. The button functions properly but the text color is the same as the background color of the popup.

**To Reproduce**
1. Add a song to a playlist
2. Click add to playlist on the same song
3. Select the same playlist as mentioned in step 1

**Expected behavior**
The text should be in a different color, preferably some color that is easily visible on the light purple background.

**Screenshots**
![image](https://user-images.githubusercontent.com/66667161/133804232-f85dad08-1a94-4ea4-bc65-808270af1b4e.png)

**Logs**

```console
vendor~xpui.js:1 MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 change listeners added. Use emitter.setMaxListeners() to increase limit
    at c (https://xpui.app.spotify.com/vendor~xpui.js:29:2656773)
    at t.o.addListener (https://xpui.app.spotify.com/vendor~xpui.js:29:2659055)
    at https://xpui.app.spotify.com/vendor~xpui.js:29:2091056
    at ka (https://xpui.app.spotify.com/vendor~xpui.js:29:2946534)
    at Ps (https://xpui.app.spotify.com/vendor~xpui.js:29:2970386)
    at t.unstable_runWithPriority (https://xpui.app.spotify.com/vendor~xpui.js:29:3190652)
    at hi (https://xpui.app.spotify.com/vendor~xpui.js:29:2916042)
    at Is (https://xpui.app.spotify.com/vendor~xpui.js:29:2970165)
    at ws (https://xpui.app.spotify.com/vendor~xpui.js:29:2954936)
    at vs (https://xpui.app.spotify.com/vendor~xpui.js:29:2953858)
    at https://xpui.app.spotify.com/vendor~xpui.js:29:2916314
    at t.unstable_runWithPriority (https://xpui.app.spotify.com/vendor~xpui.js:29:3190652)
    at hi (https://xpui.app.spotify.com/vendor~xpui.js:29:2916042)
    at yi (https://xpui.app.spotify.com/vendor~xpui.js:29:2916259)
    at vi (https://xpui.app.spotify.com/vendor~xpui.js:29:2916194)
    at As (https://xpui.app.spotify.com/vendor~xpui.js:29:2954357)
    at rl (https://xpui.app.spotify.com/vendor~xpui.js:29:2980770)
    at render (https://xpui.app.spotify.com/vendor~xpui.js:29:2983881)
    at https://xpui.app.spotify.com/xpui.js:1:1035286
(anonymous) @ vendor~xpui.js:1
vendor~xpui.js:1 cosmos-connector:  fetchAdForSlot endpoint=%o sp://ads/v1/ads/hpto
xpui.js:1 Uncaught (in promise) Error: GET request to sp://ads/v1/settings/leaderboard failed with status code 404
    at s.onSuccess (xpui.js:1)
    at onSuccess (xpui.js:1)
pubads_impl_2021091504.js:6 [GPT] Invalid argument: PubAdsService.setRequestNonPersonalizedAds("0"). Valid values: 0,1.
(anonymous) @ pubads_impl_2021091504.js:6
pubads_impl_2021091504.js:6 [GPT] Invalid argument: PubadsService.setTagForUnderAgeOfConsent("0"). Valid values: 2,0,1.
(anonymous) @ pubads_impl_2021091504.js:6
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/service-worker.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/vendor~xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui-desktop-routes-homepage-takeover-ad.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui-routes-home.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/xpui-desktop-modals.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://xpui.app.spotify.com/998.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
```
![image](https://user-images.githubusercontent.com/66667161/133804677-86306d9e-42b1-42b7-a36d-7ce2239ce270.png)

**Specifics (please complete the following information):**
 - OS: Windows 10 Home
 - Spotify version: 1.1.67.586.gbb5ef64e
 - Spicetify version: 2.6.3
 - Theme name: Ziro Dark Purple

**Comment by morpheusthewhite** [comment]
@schnensch0

**Comment by afonsojramos** [comment]
Closing this as it was fixed in #614.
